src/content/docs/open_space

```diff
@@ -0,0 +1,38 @@
+---
+title: "Open Spaces Guide"
+meta_title: "Open Spaces Guide"
+description: "Guide and Resources for Speakers"
+draft: true
+weight: 1
+---
+
+# Open Space guide for PyLadies Conference 2025
+
+## What's an Open Space?
+
+An Open Space is a dynamic, self-organized session where attendees come together to discuss topics, collaborate on projects, learn new
+skills, or just networking in a welcoming and inclusive environment.
+
+## Who can join or lead an Open Space?
+
+If you're planning to attend PyLadiesCon 2025, you're welcome to participate in or facilitate an Open Space session! This is
+your chance to share knoweldge, ask questions, or discuss about topics that intrigues you with fellow conference attendees.
+
+## How to organize an Open Space?
+
+To help us manage the logistics of the Open Spaces better, we kindly ask you to send as an email (pyladiescon@pyladies.com) if you're planning on leading one during
+the conference. We won't be approving or rejecting any Open Spaces as long as they align with our Code of Conduct.
```

Comment:
This is a little confusing,  I don't understand what it means by "we won't be approving or rejecting..." . If the open space is not accepted and not rejected,  what do we do with it?